UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -538,13 +538,14 @@ <h4>Examples</h4>
 				</section>
 
 				<section id="conf-techniques">
-					<h4>Conformance metadata techniques</h4>
+					<h4>Display techniques for conformance support</h4>
 <p>Specific techniques for meeting this principle are defined in the following documents:</p>
 
+
 					<ul>
-						<li><a href="../techniques/epub-metadata/index.html">EPUB Accessibility: Conformance</a></li>
+						<li><a href="../techniques/epub-metadata/index.html#conformance-group">EPUB Accessibility: Conformance</a></li>
 						<li><a href="../techniques/epub-metadata/index.html">EPUB Accessibility: Certification</a></li>
```

Comment:
The EPUB and ONIX certification section links are also missing fragment identifiers. Only the conformance links now have them.